### DISCUSSION TITLE
[@types/react-modal-image] Missing onClose inside Litebox Property

### DISCUSSION THREAD

**Discussion by infysumanta** [DefinitelyTyped/DefinitelyTyped#65467] [Issues with a @types package]
the type defination of the package @types/react-modal-image is not properly for Litebox, need to add onClose as a properly

[a comment by an automated account is not shown]

**Comment by hkleungai** [comment]
Hi, it may be a late reply to the question, but I have just opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71451 to address this issue. You may want to take a look.
